script/download_rust_deps.py

```diff
@@ -131,13 +131,40 @@ def parse_args():
     return args
 
 
+def cargo_install(tool):
```

Comment:
is this installing deps with cargo? I don't think this is needed even when they aren't bundled in the download cc @tmancey

Reply:
@bridiver This should not be needed, as `cbindgen` if not bundled will be downloaded and built during `npm run init`. @AndriusA let me know if you need further info. Thanks

Reply:
@tmancey didn't seem to have it working by default, vanilla setup doesn't install it by default. Could you point me to where this is setup in `npm run init`?